<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorEditor</class>
 <widget class="QTabWidget" name="ColorEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>Wheel</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="color_widgets::ColorWheel" name="colorWheel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>A:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="color_widgets::GradientSlider" name="slider_alpha_wheel">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_9">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>RGB</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>R:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="color_widgets::GradientSlider" name="slider_red">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QSpinBox" name="spinBox_5">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>G:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="color_widgets::GradientSlider" name="slider_green">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QSpinBox" name="spinBox_6">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>B:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="color_widgets::GradientSlider" name="slider_blue">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QSpinBox" name="spinBox_7">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>A:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="color_widgets::GradientSlider" name="slider_alpha_rgb">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QSpinBox" name="spinBox_8">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab_2">
   <attribute name="title">
    <string>HSV</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>H:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="color_widgets::HueSlider" name="HueSlider">
      <property name="maximum">
       <number>359</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="spinBox">
      <property name="maximum">
       <number>359</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>S:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="color_widgets::GradientSlider" name="GradientSlider">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="spinBox_2">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>V:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="color_widgets::GradientSlider" name="GradientSlider_2">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="spinBox_3">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>A:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="color_widgets::GradientSlider" name="GradientSlider_3">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QSpinBox" name="spinBox_4">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>color_widgets::ColorWheel</class>
   <extends>QWidget</extends>
   <header>color_wheel.hpp</header>
  </customwidget>
  <customwidget>
   <class>color_widgets::GradientSlider</class>
   <extends>QSlider</extends>
   <header>gradient_slider.hpp</header>
  </customwidget>
  <customwidget>
   <class>color_widgets::HueSlider</class>
   <extends>color_widgets::GradientSlider</extends>
   <header>hue_slider.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>HueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HueSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GradientSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GradientSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GradientSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GradientSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GradientSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GradientSlider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_alpha_wheel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_9</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_9</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_alpha_wheel</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_red</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_red</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_green</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_green</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_blue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_blue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_alpha_rgb</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_alpha_rgb</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
